pkg/tar/tar.go

```diff
@@ -215,6 +222,22 @@ func (t *stiTar) ExtractTarStream(dir string, reader io.Reader) error {
 	}
 }
 
+func extractLink(dir string, header *tar.Header, tarReader io.Reader) error {
+	dest := filepath.Join(dir, header.Name)
+	source := header.Linkname
+
+	glog.V(3).Infof("Creating symbolic link from %s to %s", dest, source)
+
+	if err := os.Symlink(source, dest); err != nil {
```

Comment:
just `return os.Symlink(source, dest)` here, no need for the glog below (or if you really want to do glog, make it as `defer`